app/lib/bk/compat/parsers/gha/expressions.rb

```diff
@@ -154,6 +166,16 @@ def self.replace_context_needs(path)
         end
       end
 
+      def self.replace_context_github(var_name)  
+        translated_var = @github_bk_context_mapping[var_name]
+        case translated_var
+        when String
+          "$#{translated_var}"
```

Comment:
based on previous reviews and implementations this is missing a `$` to avoid escaping on pipeline upload:
```suggestion
          "$$#{translated_var}"
```

Reply:
Switched to the `raise``return method vs the case as above - including the dual `$$` 👆